packages/mobile/src/identity/matchmaking.ts

```diff
@@ -12,12 +12,16 @@ import { NumberToRecipient } from 'src/recipients/recipient'
 import Logger from 'src/utils/Logger'
 import { getAuthSignerForAccount } from 'src/web3/dataEncryptionKey'
 import { getConnectedUnlockedAccount } from 'src/web3/saga'
+import { mtwAddressSelector } from 'src/web3/selectors'
 
 const TAG = 'identity/matchmaking'
 
 // Uses the phone number privacy service to find mutual matches between Celo users
 export function* fetchContactMatches(e164NumberToRecipients: NumberToRecipient) {
-  const account: string = yield call(getConnectedUnlockedAccount)
+  const walletAddress: string = yield call(getConnectedUnlockedAccount)
```

Comment:
Should these be of type `Address`?

Reply:
We use `string` instead of `Address` in most places across Valora. I agree we should be better about using more specific typing.